File: app/src/main/java/dev/studiocloud/instamovie/data/services/response/movieResponse/MovieResponse.java
package dev.studiocloud.instamovie.data.services.response.movieResponse;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MovieResponse {

    @SerializedName("page")
    private int page;

    @SerializedName("total_pages")
    private int totalPages;

    @SerializedName("results")
    private List<MovieItem> results;

    @SerializedName("total_results")
    private int totalResults;

    public void setPage(int page) {
        this.page = page;
    }

    public int getPage() {
        return page;
    }

    public void setTotalPages(int totalPages) {
        this.totalPages = totalPages;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public void setResults(List<MovieItem> results) {
        this.results = results;
    }

    public List<MovieItem> getResults() {
        return results;
    }

    public void setTotalResults(int totalResults) {
        this.totalResults = totalResults;
    }

    public int getTotalResults() {
        return totalResults;
    }

    @Override
    public String toString() {
        return
                "MovieResponse{" +
                        "page = '" + page + '\'' +
                        ",total_pages = '" + totalPages + '\'' +
                        ",results = '" + results + '\'' +
                        ",total_results = '" + totalResults + '\'' +
                        "}";
    }
}

File: app/src/main/java/dev/studiocloud/instamovie/data/remote/response/movieResponse/MovieResponse.kt
package dev.studiocloud.instamovie.data.remote.response.movieResponse

import com.google.gson.annotations.SerializedName

class MovieResponse {
    @SerializedName("page")
    var page = 0

    @SerializedName("total_pages")
    var totalPages = 0

    @SerializedName("results")
    var results: MutableList<MovieItem>? = null

    @SerializedName("total_results")
    var totalResults = 0
    override fun toString(): String {
        return "MovieResponse{" +
                "page = '" + page + '\'' +
                ",total_pages = '" + totalPages + '\'' +
                ",results = '" + results + '\'' +
                ",total_results = '" + totalResults + '\'' +
                "}"
    }
}